Добавил README.md с описанием функциональных и нефункциональных требований системы и расчетом нагрузки.

Tarasusrus opened the pull request:
Что внутри:
	•	Описание ключевых фич (посты, комментарии, реакции, подписки, поиск, лента).
	•	Нефункциональные требования (SLA, мультиязычность, сезонность, лимиты, рост DAU).
	•	Подсчет RPS и трафика по подсистемам (лента, посты, комментарии, реакции).
	•	Таблицы с объемами данных и API-трафиком.
	
Зачем:

Это базовый артефакт для дальнейшего проектирования — описывает, что именно строим и какую нагрузку ожидаем.

Tarasusrus committed:
Added initial draft of `README.md` with functional and non-functional requirements, system load estimation, and JSON schemas.

Tarasusrus committed:
Update README.md: add entity weight calculation section

BalunCourses commented on README.md:
```diff
@@ -0,0 +1,152 @@
+    4. (Данные) Данные храним всегда
+    5. (Лимиты) Нет лимитов на подписки. Публикация поста, комментарий, лойк, подписка отрабатвают в течении 5с.
+    6. (Доступность) 99.9 - 8.7 ч
+    7. (Connections) = dau \* 0.1 = 1 000 000
```
Это уже не требования вместе с RPS

BalunCourses commented on README.md:
```diff
@@ -0,0 +1,152 @@
+    5. (Лимиты) Нет лимитов на подписки. Публикация поста, комментарий, лойк, подписка отрабатвают в течении 5с.
+    6. (Доступность) 99.9 - 8.7 ч
+    7. (Connections) = dau \* 0.1 = 1 000 000
+    8. RPS = (10 000 000×5 + 100 000×1 + 900 000×2 + 900 000×5) / 86 400 = ~653
```
Что здесь считается - непонятно

BalunCourses commented on README.md:
```diff
@@ -0,0 +1,152 @@
+    3. (Сезонность) Ожидается рост активности в летний период(коэффициент сезонного пика ×1.5–2.)
+    4. (Данные) Данные храним всегда
+    5. (Лимиты) Нет лимитов на подписки. Публикация поста, комментарий, лойк, подписка отрабатвают в течении 5с.
+    6. (Доступность) 99.9 - 8.7 ч
```
А проценты где?)

BalunCourses commented on README.md:
```diff
@@ -0,0 +1,152 @@
+    0.09 · 10 000 000 · 2 = 1 800 000 => 1 800 000 / 86 400 = 20.833 RPS
+- Реакции
+   0.09 · 10 000 000 · 5 = 4 500 000 => 4 500 000 / 86 400 = 52.083 RPS
+##### Трафик по компонентам
```
Нужна считать в разрезе трафик по медиа и метаинформации

BalunCourses commented on README.md:
```diff
@@ -0,0 +1,152 @@
+После этого, нужно будет примерно **оценить нагрузку** (RPS и трафик) на будущую систему. 
+**Оценку нагрузки рекомендуется проводить в разрезе по подсистемам** (*посты, реакции, комментарии*) вашей будущей системы.
+
+### Нефункциональные требования
```
А тайминги и сезонность?

Tarasusrus committed:
- Revise README to clarify functional and non-functional requirements
- Align user behavior metrics and RPS calculations
- Format tables for consistency and readability

Tarasusrus committed:
Simplify README by reorganizing sections, removing redundancy, and adding structured FR/NFR details with load estimations.
